_includes/card.html

```diff
@@ -8,21 +8,22 @@
   {% assign wds_grid_class = 'usa-width-one-third' %}
 {% endif %}
 
-{% assign project_type = include.project_type | default: 'none' %}
-
-<article class="card {{ wds_grid_class }}" data-bucket="{{ project_type }}">
-  <a class="card-link" href="{{ card_link }}">
-    <div role="img" title="{{ include.image_alt }}" class="card-image-bg" style="background-image: url({{ site.baseurl }}{{ include.image_src }});">
-    </div>
+<article class="card {{ wds_grid_class }}">
+  <div class="card-link" onclick='window.location = "{{ include.link }}";' tabindex="-1">
+    <div role="img"
+         title="{{ include.image_alt }}"
```

Comment:
I believe I've addressed this in `9c36a35`, but we should make sure this is noted elsewhere as an accessibility training resource, since I know people are often writing alt-text on the fly (not just for this site, but across 18F).